### ISSUE TITLE
FieldSet in documentation does not exist in package

### ISSUE THREAD

**Issue by wz2b** [grafana/grafana#26127]
[This Documentation](https://developers.grafana.com/ui/latest/index.html?path=/docs/forms-fieldset--basic) shows that there should be a FieldSet in @grafana/ui but there doesn't seem to be such a thing exported.  

```
  "devDependencies": {
    "@grafana/data": "^7.0.5",
    "@grafana/runtime": "^7.0.5",
    "@grafana/toolkit": "^7.0.5",
    "@grafana/ui": "^7.0.5",
    ...
```

**Comment by Clarity-89** [comment]
Hi, the `FieldSet` component is available starting version 7.1. Seems like the documentation is out of sync there.

**Comment by wz2b** [comment]
Awesome, thanks.
